include/bitcoin/blockchain/interface/safe_chain.hpp

```diff
@@ -120,6 +120,12 @@ class BCB_API safe_chain
     virtual void fetch_block_height(const hash_digest& hash,
         block_height_fetch_handler handler) const = 0;
 
+    void fetch_block_txs_size(const hash_digest& hash,
```

Comment:
I don't like it, but we have to be consistent with the existing member functions: pure virtual